Cargo.toml

```diff
@@ -15,10 +15,20 @@ readme = "README.md"
 repository = "https://github.com/datafusion-contrib/datafusion-federation"
 
 [workspace.dependencies]
-async-trait = "0.1.81"
+arrow = "53.0.0"
+arrow-flight = { version = "53.0.0", features = ["flight-sql-experimental"] }
+arrow-json = "53.0.0"
 async-stream = "0.3.5"
+async-trait = "0.1.81"
+datafusion = "42.0.0"
+# XXX use the release version on crates.io
```

Comment:
Could we use the crate version from crates.io here instead of the path?

Reply:
Yes, I think we should do both, otherwise the code will not be able to build locally as the server and table provider crates depend on the federation crate. I had to look it up to refresh my memory, see https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#local-paths-in-published-crates. In short when using both path and version, cargo uses the path when building locally and the version when publishing to the registry.